syntax/hcl.vim

```diff
@@ -32,7 +32,7 @@ syn match hclNumber /\<0[xX]\x\+\>/
 
 syn keyword hclConstant true false null
 
-syn region hclInterpolation start=/\${/ end=/}/ contained contains=hclInterpolation
+syn region hclInterpolation start=/\${/ end=/}/ contained contains=hclString,hclInterpolation,hclAttribute,hclAttributeName
```

Comment:
Do you have an example that has a string inside an interpolation? I admit I work almost exclusively with Terraform, but I have never seen such a construct in the wild. 🙂 

In similar vein, is there a use case, which utilizes an `hclAttribute` (`/^.*=/`) inside an interpolation? 🤔

Finally, I think possibly the addition of `hclAttributeName` here breaks highlighting of variables (i.e. identifiers) inside interpolations, as they now become unadorned (i.e. light gray) instead of being highlighted the same way `hclVariable` (i.e. yellow). I think this is confusing, because it changes how variables are highlighted inside and outside interpolations.